includes/definitions/cyberpower.yaml

```diff
@@ -1,13 +1,13 @@
 os: cyberpower
-text: Cyberpower
+text: 'Cyberpower PDU'
 type: power
 icon: cyberpower
 over:
     - { graph: device_current, text: Current }
     - { graph: device_voltage, text: Voltage }
 discovery:
     - sysObjectID:
-        - .1.3.6.1.4.1.3808
+        - .1.3.6.1.4.1.3808.1.1.3
```

Comment:
This is probably not a good idea. You should exclude ```1.3.6.1.4.1.3808.1.1.1``` from the existing cyberpower (because you handle it into the new cyberpower-ups.yaml) and leave the all the other products to be matched by cyberpower.yaml.

Reply:
@PipoCanaja  Ah, excellent point. I'll revert my previou changes and add the exception